client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml

```diff
@@ -463,18 +463,18 @@
    <translation id="privateDatabase_order_sql_type_premium_label" qtlid="72845">SQL Privé</translation>
    <translation id="privateDatabase_order_sql_type_premium_info_title" qtlid="353491">Points forts des SQL Privé :</translation>
    <translation id="privateDatabase_order_sql_type_premium_info_line_1" qtlid="404811">Nombre de bases illimité</translation>
-   <translation id="privateDatabase_order_sql_type_premium_info_line_2" qtlid="353517">Mémoire RAM garantie de 256 Mo à 2 Go</translation>
+   <translation id="privateDatabase_order_sql_type_premium_info_line_2">Mémoire RAM garantie de 512 Mo à 4 Go</translation>
```

Comment:
Can we upate this translation to be dynamic by using values returned by API ? :D

Reply:
Done ;)